catalyst_voices/apps/voices/integration_test/app_test.dart

```diff
@@ -1,20 +1,59 @@
 import 'package:catalyst_voices/app/view/app.dart';
 import 'package:catalyst_voices/configs/bootstrap.dart';
+import 'package:catalyst_voices/routes/routes.dart';
+import 'package:catalyst_voices_models/catalyst_voices_models.dart';
 import 'package:flutter_test/flutter_test.dart';
 import 'package:integration_test/integration_test.dart';
+import 'pageobject/dashboard_page.dart';
+import 'pageobject/spaces_drawer_page.dart';
 
 void main() {
   IntegrationTestWidgetsFlutterBinding.ensureInitialized();
 
+  tearDownAll(() async {
+    // enable to make sure chrome window stays open after test to check console prints or something
+    // await Future<void>.delayed(const Duration(minutes: 5));
```

Comment:
Do we really need to keep this in the code?

Reply:
Absolutely not, it was meant for convenience for beginners like me ;)
Removed